README.md

```diff
@@ -17,7 +17,7 @@ The environment can be reproduced using the following commands:
 ```
 python3 -m venv my_virtual_environment
 source my_virtual_environment/bin/activate
-pip3 -r requirements.txt
+pip3 install -r requirements.txt
```

Comment:
This file is generated from other files. You also need to change intro.md so this change isn't overwritten

Reply:
Done in [018e66f](https://github.com/pyccel/pyccel-benchmarks/pull/14/commits/018e66f391048146527cfc699c8884a4c6b00ab6)